pyvcloud/schema/vcd/v1_5/schemas/vcloud/networkType.py

```diff
@@ -13860,12 +13860,12 @@ def buildChildren(self, child_, node, nodeName_, fromsubclass_=False):
     'ParentNetwork': ReferenceType,
     'References': ReferencesType,
     'SubAllocations': SubAllocationsType,
-    'AllocatedIpAddresses': IpAddressesType,
+    'AllocatedIpAddresses': AllocatedIpAddressType,
     'PeerSubnet': IpsecVpnSubnetType,
     'ServiceProfile': LBVirtualServerServiceProfileType,
     'StaticRoutingService': StaticRoutingServiceType,
     'VirtualServer': LoadBalancerVirtualServerType,
-    'IpAddress': AllocatedIpAddressType,
+    'IpAddress': IpAddressesType,
```

Comment:
IpAddress is also defined as IpAddress and as AllocatedIpAddressType. If we change to IpAddressType (Singular) we need to validate that other parts of the code don't break.